inference-engine/tests/functional/shared_test_classes/src/single_layer/low_precision.cpp

```diff
@@ -0,0 +1,92 @@
+// Copyright (C) 2019-2021 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include "shared_test_classes/single_layer/low_precision.hpp"
+#include "ngraph_functions/builders.hpp"
+
+namespace LowPrecisionTestDefinitions {
+
+std::string LowPrecisionTest::getTestCaseName(testing::TestParamInfo<lowPrecisionTestParamsSet> obj) {
+    InferenceEngine::Precision netPrecision;
+    std::string targetDevice;
+    std::pair<std::string, std::map<std::string, std::string>> config;
+    std::tie(netPrecision, targetDevice, config) = obj.param;
+
+    std::ostringstream result;
+    result << "netPRC=" << netPrecision.name() << "_";
+    result << "trgDev=" << targetDevice;
+    if (!config.first.empty()) {
+        result << "_targetConfig=" << config.first;
+    }
+    return result.str();
+}
+
+void LowPrecisionTest::SetUp() {
+    InferenceEngine::Precision netPrecision;
+    std::pair<std::string, std::map<std::string, std::string>> config;
+    std::tie(netPrecision, targetDevice, config) = this->GetParam();
+    auto ngPrc = FuncTestUtils::PrecisionUtils::convertIE2nGraphPrc(netPrecision);
+    auto inputShape = ngraph::Shape{ 1, 16 };
+    auto weights1Shape = ngraph::Shape{ 16, 16 };
+    auto weights2Shape = ngraph::Shape{ 128, 32 };
+
+    // fully connected 1
+    auto input = std::make_shared<ngraph::opset1::Parameter>(ngPrc, inputShape);
+    std::vector<float> weights1Data(ngraph::shape_size(weights1Shape), 0.0f);
+
+    auto count = 10;
+    for (int i = 0; i < 256; i+=17) {
+        weights1Data[i] = count++;
+    }
```

Comment:
for (int i = 0; i < 16; i++) {
        weights1Data[i*17] = i;
    }

Reply:
Done.